draft-demarco-nonce-endpoint.md

```diff
@@ -159,37 +156,37 @@ Please note that the values represented in the previous examples may depend on d
 
 The Nonce Endpoint MUST be protected by TLS to prevent eavesdropping and man-in-the-middle attacks, therefore the practices defined in [BCP195] should be followed.
 
-The Nonce Issuer MUST securely generate and store the encryption key used to encrypt the Nonce. The robustness of the encryption key plays a crucial role in the security of the Nonce Endpoint. The following considerations should be taken into account:
+The Nonce Issuer MUST securely generate and store the encryption key used to encrypt the Nonce. The robustness of the encryption key plays a crucial role in the security of the Nonce Endpoint. The following considerations MUST be taken into account:
```

Comment:
in the consideration we don't require normative terms like MUST and so on ... just FYI